.github/workflows/release-build-all.yml

```diff
@@ -0,0 +1,199 @@
+      - name: Build SIMS-API Image
+        working-directory: "./devops"
+        run: |
+          make oc-build-api
+
+  # Build Workers
+  build-workers:
+    name: Build Workers
+    runs-on: ubuntu-latest
+    needs: createTag
+    env:
+      BUILD_REF: ${{ needs.createTag.outputs.newTag }}
+    steps:
+      - name: Print env
+        run: |
+          echo BUILD NAMESPACE: $BUILD_NAMESPACE
+          echo BRANCH: ${{ needs.createTag.outputs.newTag }}
+      - name: Checkout Target Branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.createTag.outputs.newTag }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
+      - name: Build Workers Image
+        working-directory: "./devops"
+        run: |
+          make oc-build-workers
+
+  # Build Queue Consumers
+  build-queue-consumers:
+    name: Build Queue Consumers
+    runs-on: ubuntu-latest
+    needs: createTag
+    env:
+      BUILD_REF: ${{ needs.createTag.outputs.newTag }}
+    steps:
+      - name: Print env
+        run: |
+          echo BUILD NAMESPACE: $BUILD_NAMESPACE
+          echo BRANCH: ${{ needs.createTag.outputs.newTag }}
+      - name: Checkout Target Branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.createTag.outputs.newTag }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
+      - name: Build Queue Consumers Image
+        working-directory: "./devops"
+        run: |
+          make oc-build-queue-consumers
+
+  # Build Web/Frontend
+  build-web-frontend:
+    name: Build Web/Frontend
+    runs-on: ubuntu-latest
+    needs: createTag
+    env:
+      BUILD_REF: ${{ needs.createTag.outputs.newTag }}
+    steps:
+      - name: Print env
+        run: |
+          echo BUILD NAMESPACE: $BUILD_NAMESPACE
+          echo BRANCH: ${{ needs.createTag.outputs.newTag }}
+      - name: Checkout Target Branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.createTag.outputs.newTag }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
+      - name: Build Web/Frontend Image
+        working-directory: "./devops"
+        run: |
+          make oc-build-web
+
+  # Deploy all applications to DEV when there is a pull request on the main branch.
+  # If DEV is used to deploy a release or hotfix branch the automatic deploy can be stopped using the env variable because there
+  # is a potencial to main/relase/hotfix branches have differences (for instance o DB) that would not be backward compatible.
+  deployAll:
+    if: github.event_name == 'pull_request' && vars.ALLOW_AUTOMATIC_MAIN_BRANCH_DEPLOY_TO_DEV == 'true' && github.ref_name == 'main'
```

Comment:
for my info, I tried looking up `ALLOW_AUTOMATIC_MAIN_BRANCH_DEPLOY_TO_DEV` on github but couldn't find it. Where can find this?

Reply:
It is needed admin access, not every DEV has it.
It would be here
![image](https://github.com/bcgov/SIMS/assets/61259237/b6d510b5-ccb4-4066-9a3e-f483478db751)

Reply:
Thanks